container-images/pragmatic/build_pragmatic.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+clone_and_build_pragmatic() {
```

Comment:
We should add

set -exu -o pipefail

We want the build to fail if these commands fail.

Reply:
Ok, I am still working on making the install smaller.  Gotten CPU cut in 1/3 but little change in ROCM and CUDA.

Reply:
Should we add the same flag to build_llama_and_whisper.sh?

Reply:
`set -ex` is there, it's just in the main function (we could add, `-o pipefail`, etc. if it suits). I tend to create main functions in shell scripts to encourage using scope and less global variable.

`set -e` alone covers a lot to be fair sometimes it's sufficient.

I do like to add `-x` at times also, makes CI easy to debug.

But `set -e` at least is a good idea.